### PR TITLE
En 10337 full history fixes

### PR DESCRIPTION
**Pull request by raduchis**
Fixed logging of storage information and files open. 
Fixed restart of full history in epoch 0.
Fixed eviction of old pruning storers for full archive nodes.

### CODE REVIEW

**Review by iulianpascalau — DISMISSED** [review]

File: storage/latestData/fullHistoryLatestDataProvider.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+func (ldp *fullHistoryLatestDataProvider) Get() (storage.LatestDataFromStorage, error) {
+	epochDirs, err := ldp.getEpochDirs()
+	if err != nil {
+		return storage.LatestDataFromStorage{}, err
+	}
+
+	for index := range epochDirs {
+		parentDir, lastEpoch, errGetDir := ldp.getParentDirAndLastEpochWithIndex(index)
+		if errGetDir != nil {
+			err = errGetDir
```
[the remark this answers is not in the document]
Comment (raduchis) [reply storage/latestData/fullHistoryLatestDataProvider.go:32]: fixed

File: storage/latestData/fullHistoryLatestDataProvider.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+func (ldp *fullHistoryLatestDataProvider) GetParentDirAndLastEpoch() (string, uint32, error) {
+	epochDirs, err := ldp.getEpochDirs()
+	if err != nil {
+		return "", 0, err
+	}
+
+	for index := range epochDirs {
+		parentDir, lastEpoch, errGetDir := ldp.getParentDirAndLastEpochWithIndex(index)
+		if errGetDir != nil {
+			continue
```
[the remark this answers is not in the document]
Comment (raduchis) [reply storage/latestData/fullHistoryLatestDataProvider.go:58]: fixed

File: storage/latestData/fullHistoryLatestDataProvider.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	}
+
+	for index := range epochDirs {
+		parentDir, lastEpoch, errGetDir := ldp.getParentDirAndLastEpochWithIndex(index)
+		if errGetDir != nil {
+			err = errGetDir
+		}
+
+		latestDataFromStorage, errGetEpoch := ldp.getLastEpochAndRoundFromStorage(parentDir, lastEpoch)
+		if errGetEpoch == nil {
```
[the remark this answers is not in the document]
Comment (raduchis) [reply storage/latestData/fullHistoryLatestDataProvider.go]: fixed

**Review by SebastianMarian — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
System tests passed

**Pull request merged**